test/unit/webui/components/__mocks__/api.js

```diff
@@ -26,7 +27,13 @@ const register = (url, method = 'get', options = {}) => {
     });
   }
 
-  throw Error('Not found');
+  if (url === 'packages' && method.toLocaleLowerCase() === 'get') {
```

Comment:
hahaha... Okay.

Reply:
Tip:  I learned this in Getify's workshop. 

We should limit the use of arrow functions. It indicates lexical scoping of this and if we will use arrow functions everywhere it might give the wrong indication to the reader of the code.  

I know we (@verdaccio/core-team ) are using these arrow functions everywhere (in this PR as well) but we can try to limit the use of them in the future.

Reply:
Some notes:

https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20%26%20beyond/ch2.md#arrow-functions

> So now we can conclude a more nuanced set of rules for when => is appropriate and not:
> If you have a short, single-statement inline function expression, where the only statement is a return of some computed value, and that function doesn't already make a this reference inside it, and there's no self-reference (recursion, event binding/unbinding), and you don't reasonably expect the function to ever be that way, you can probably safely refactor it to be an => arrow function.
> If you have an inner function expression that's relying on a var self = this hack or a .bind(this) call on it in the enclosing function to ensure proper this binding, that inner function expression can probably safely become an => arrow function.
If you have an inner function expression that's relying on something like var args = Array.prototype.slice.call(arguments) in the enclosing function to make a lexical copy of arguments, that inner function expression can probably safely become an => arrow function.
For everything else -- normal function declarations, longer multistatement function expressions, functions that need a lexical name identifier self-reference (recursion, etc.), and any other function that doesn't fit the previous characteristics -- you should probably avoid => function syntax.

https://nodejs.org/api/events.html#events_passing_arguments_and_this_to_listeners
> It is possible to use ES6 Arrow Functions as listeners, however, when doing so, the this keyword will no longer reference the EventEmitter instance:

Reply:
@juanpicado @ayusharma interesting!